console/console-init/ui/src/modules/address-space/components/AddressSpaceNavigation/AddressSpaceNavigation.tsx

```diff
@@ -46,6 +46,19 @@ export const AddressSpaceNavigation: React.FunctionComponent<AddressSpaceNavigat
             Connections
           </NavLink>
         </NavItem>
+        <NavItem
+          key="endpoints"
+          itemId="endpoints"
+          isActive={active === "endpoints"}
+        >
+          <NavLink
+            id="ad-space-nav-endpoints"
+            to={`endpoints`}
+            style={{ color: "black" }}
```

Comment:
is it possible to use css class instead of inline css and use patternfly color code if possible.

Reply:
The inline style here is applied to all the `NavLinks` we can have a common class here.